extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.js

```diff
@@ -239,9 +247,23 @@ function _checkIfCanAddMeasurementsToDisplaySet(
   }
 }
 
-function _measurementReferencesSOPInstanceUID(measurement, SOPInstanceUID) {
+function _measurementReferencesSOPInstanceUID(
+  measurement,
+  SOPInstanceUID,
+  frameNumber
+) {
   const { coords } = measurement;
 
+  // NOTE: The ReferencedFrameNumber can be multiple values according to the DICOM
+  //  Standard. But for now, we will support only one ReferenceFrameNumber.
+  const ReferencedFrameNumber =
+    (measurement.coords[0].ReferencedSOPSequence &&
+      measurement.coords[0].ReferencedSOPSequence.ReferencedFrameNumber) ||
```

Comment:
You should use ReferencedSOPSequence[0] - it is an array that happens to have attributes of child zero when of length 1,  but you shouldn't count on that.

Reply:
fixed in c48cbff4fd68e6848196fec07d0b49b7bffe0fd8